docs/readme.md

```diff
@@ -1,4 +0,0 @@
-# Documentation
```

Comment:
This README file is here because of an external user who thought we had no docs because they found a nearly empty `docs` folder in our repo. I'd be happy to change it (e.g. to point to [docs.rs/risc0-zkvm](docs.rs/risc0-zkvm)) but I don't think we should drop it entirely.

Reply:
Thanks, I'll drop it for now but plan to deal with this in the future.